docs/design/modules/digital_twin_graph/diagrams/find_sequence.puml

```diff
@@ -0,0 +1,69 @@
+@startuml
+
+autonumber
+
+participant "Digital Twin Consumer" as CONSUMER
+participant "Digital Twin Graph" as DIGITAL_TWIN_GRAPH
+participant "Digital Twin Registry" as DIGITAL_TWIN_REGISTRY
+participant "Digital Twin Provider" as PROVIDER
+
+CONSUMER -> DIGITAL_TWIN_GRAPH: Find(model_id: "dtmi:sdv:seat;1") - request
+
+DIGITAL_TWIN_GRAPH -> DIGITAL_TWIN_REGISTRY: FindByModeld(model_id: "dtmi:sdv:seat;1") - request
+
+DIGITAL_TWIN_GRAPH <- DIGITAL_TWIN_REGISTRY: FindByModelId - response
+note left
+  list of EndpointInfo
+
+  [
+    {
+      model_id : "dtmi:sdv:seat;1"
+      instance_id: "front left seat"
+      protocol: "grpc"
+      operations: ["get", "invoke"]
+      uri: Digital Twin Provider's uri
+    },
+    {
+      model_id : "dtmi:sdv:seat;1"
+      instance_id: "front right seat"
+      protocol: "grpc"
+      operations: ["get", "invoke"]
+      uri: Digital Twin Provider's uri
+    }
+  ]
+end note
+
+loop Iterate over the results from the FindByModelId call
+  DIGITAL_TWIN_GRAPH -> PROVIDER: Ask(respond_uri: respond uri for Digital Twin Graph, ask_id: "1", payload: {instance_id: "front left seat", operation: "get" })
+  DIGITAL_TWIN_GRAPH <- PROVIDER: Answer(ask_id: "1", payload: instance value as JSON-LD string)
+end
+
+CONSUMER <- DIGITAL_TWIN_GRAPH: Find - response
+note left
+  list of instance values as JSON-LD string
```

Comment:
based on the code I'm seeing, this response should include the payload from the answer. Is the example in this note up to date?

Reply:
The diagram is correct.  Let's go over this together, as I think that there is a misunderstanding here that needs to be addressed.

Reply:
I checked the diagram again with WIlliam and we agreed that it is fine.